lib/matplotlib/testing/jpl_units/UnitDblFormatter.py

```diff
@@ -36,12 +36,12 @@ def __call__( self, x, pos = None ):
       if len(self.locs) == 0:
          return ''
       else:
-         return str(x)
+         return '{:.12g}'.format(x)
```

Comment:
Using ".12" instead of ".12g" would keep the ".0" in the test images, and perhaps avoid the need to change some of them?

Reply:
You might be right, but let's see how the nightly build feels about it.